fontforge/encoding.c

```diff
@@ -498,12 +498,19 @@ static char *getPfaEditEncodings(void) {
     return encfile;
 }
 
-static void EncodingFree(Encoding *item) {
+void EncodingFree(Encoding *item) {
+/* TODO: Fix error if repeatedly freed more than 2x */
```

Comment:
I don't think that we need to do that in this function. The caller ought to set the pointer of a freed object to NULL.